articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Azure AD v2 JS SPA Quickstart | Microsoft Docs
+description: How JavaScript SPA applications can call an API that require access tokens by Azure Active Directory v2 endpoint
+services: active-directory
+documentationcenter: dev-center-name
+author: andretms
+manager: mbaldwin
+editor: ''
+
+ms.service: active-directory
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: identity�
+ms.date: 05/09/2018
+ms.author: andret
+
+---
+
+# Sign-in users and acquire an access token from a JavaScript application
+
+This quickstart contains a code sample that demonstrates how a JavaScript Single Page Application (SPA) can sign in personal, work and school accounts, get an access token, and call the Microsoft Graph API.
+
+![How the sample app generated by this Quickstart works](media/active-directory-javascriptspa/javascriptspa-intro.png)
+
+> [!div renderon="docs"]
+> ## Register your application and download your Quickstart app
+> You have two options to start your Quickstart application:
+> ### Option 1: Register your application and download your Quickstart app [Express]
+>
+> 1. Go to the [Azure Portal - Application Registration [Prod]](https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/JavascriptSpaQuickstartPage/sourceType/docs)
+> 1. Enter a name for your application and click **Register**
+> 1. Follow the instructions to download and configure your new application, adding **http<span/>://localhost:30662/** as Redirect URL and setting the **Implict Flow** automatically for you.
```

Comment:
This sentence is confusing.

Reply:
@manoj-rath - I changed the sentence - please check